content/departments/engineering/product/process/index.md

```diff
@@ -38,6 +38,31 @@ See [strategy](../../../../strategy-goals/index.md).
 - [Pricing and tiering/packaging](gtm/pricing.md)
 - [Licensing](gtm/licensing.md)
 
+#### Experimenting on sourcegraph.com
+
+We welcome experimentations on sourcegraph.com. It should be very easy to experiment on .com, send a tweet or email campaign out to get some traffics, and get early feedback in.
+
+The bar for an experiment on .com is the following:
+
+- Break nothing (new UIs and public data APIs are low risk, anything involving sensitive data or critical APIs is high risk)
+- Don't disrupt the main flows
+- Add basic usage metrics to know what's going on
+
+In particular an experiment does not require:
+
+- Discoverability
+- A polished design
+- Approvals, as long as it meets the criteria above
+- A/B testing, although A/B testing can be used
+
+When you experiment on .com you commit to:
+
+- Make a small announcement in #progress when you start the experiment
```

Comment:
Should we ask the user to let @sourcegraph/dev-experience know they are launching new functionality on an instance they manage?

Reply:
Good question. @chrispine ?

Reply:
Yes please, that would be great. A mention in #ask-dev-experience should suffice